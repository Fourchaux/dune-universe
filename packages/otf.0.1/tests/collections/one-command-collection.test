PROG=./otf
ARGS=-run-collection tests/one-command
EXIT_CODE=1
