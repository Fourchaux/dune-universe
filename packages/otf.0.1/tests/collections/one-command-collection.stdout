================================================================================
Running collection "tests/one-command"
4 tests collected

false-command .
true-command .
wrong-false-command F
wrong-true-command F

================================================================================
------ tests/one-command/wrong-false-command -----------------------------------
Invalid exit code (got 1 while expecting 0)

------ tests/one-command/wrong-true-command ------------------------------------
Invalid exit code (got 0 while expecting 1)

================================================================================
Summary: 2 passed, 2 failed
================================================================================
